pkg/kubectl/scale.go

```diff
@@ -79,8 +80,13 @@ const (
 	ScaleGetFailure ScaleErrorType = iota
 	ScaleUpdateFailure
 	ScaleUpdateInvalidFailure
+	// AlreadyScaled is not really an error but we need a way to surface to the client that
+	// the scaling didn't happen because we already have the desired state the user asked for.
+	AlreadyScaled
 )
 
+var alreadyScaledErr = goerrors.New("desired replicas equals with the current state")
```

Comment:
"desired replicas already equals the requested replicas"? I realize now we're not comparing to "actual"